Automattic/jetpack
# Use the protect-status package in the Protect plugin

In this PR we are implementing use of the protect-status package that was created in https://github.com/Automattic/jetpack/pull/37894.  Additionally, this PR can serve as a method of testing PR https://github.com/Automattic/jetpack/pull/37894.





## Proposed changes:

* Remove all occurrences of any protect-status related code.
* Require and use the `protect-status` package.

### Other information:


## Jetpack product discussion



This PR is part of the PT: **Add value to the active Protect product card on My Jetpack project**: pbNhbs-aP6-p2

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Create a Jurassic Ninja site with the Jetpack Beta Plugin, running Protect branch `add/protect-status-pkg-to-plugin`.
* Connect Jetpack
* Activate the Protect plugin and verify everything is working fine as expected, and with no errors.
